### DISCUSSION TITLE
Err: (InvalidArgument) The type of data we are trying to retrieve does not match the type of data cu

### DISCUSSION THREAD

**Discussion by lijiong20** [PaddlePaddle/PaddleSpatial#246] [BaiduKDDCup2022]
I passed local test but got this error online
please help
![image](https://user-images.githubusercontent.com/82355828/177977485-21742fde-726c-433c-8734-fc9e06daee9f.png)

**Comment by ramber1836** [comment]
The team name of your submission should be provided so that I can check the error log for you.
